### DISCUSSION TITLE
can not bind default value to date

### DISCUSSION THREAD

**Discussion by sqllyw** [alpinejs/alpine#1119] [1. Help] [answered]
Hi,
got a simple code like this:

https://jsfiddle.net/ghyujvm0/8/

dateStart and dateEnd are initialized in the x-data, but it did not got displayed in the input, and when click, the date picker is default to the current date, not the date set in the x-data, any idea why? Thanks

**Comment by KevinBatdorf** [comment, marked as answer]
That input type requires specific formatting, yyyy-mm-dd. You can get it with: 

```
new Date('2010-08-22T12:01:48.000Z').toISOString().split('T')[0]
```

Also, you don't need `x-model` and `:value` on the same element. `x-model` is short for `:value` and `@input`

**Reply by sqllyw** [reply]
thanks, i did not know this until now, working version as follow:
https://jsfiddle.net/ghyujvm0/12/
